src/app/cli/styx.sh

```diff
@@ -368,10 +368,7 @@ fi
 if [ "$action" = new ] && [ "$newCommand" = site ]; then
   target="$in/$name"
   check_dir "$target" "Error: Cannot create a new site in '$target', directory exists."
-  mkdir "$target"
-  mkdir "$target"/{themes,data}
-  cp -r "$newsite"/* "$target/"
-  chmod -R u+rw "$target"
+  $nix flake new "$target" --template "$root#default"
```

Comment:
If you see this and wonder:

The derivation ships with the entire source tree overlayed onto the root of the derivation.